app.py

```diff
@@ -5,6 +5,14 @@
 app = Flask(__name__)
 
 
+PROPERTY_TYPES = {
```

Comment:
Down the line, this could just be "the last part of the URL" instead of defining each one in a dictionary? :)